containers/podman-desktop
# feat: Add Minikube extension

### What does this PR do?

This PR adds a "minikube" extension, for creating a Kubernetes cluster - similar to the existing "kind" extension

### Screenshot/screencast of this PR

![podman-desktop-minikube-create](https://github.com/containers/podman-desktop/assets/10364051/5c2ab269-82c9-4421-b6c0-e026fdd9f05c)

### What issues does this PR fix or reference?

* #2610

### How to test this PR?

Create a new minikube cluster container, using the Resources and the "Create new..." button.

There should be an option to install the `minikube` binary, if it is not already installed on the host.

The docs can go in a separate PR, it would be similar to https://podman-desktop.io/docs/kubernetes/kind

Note for reviewer: this "minikube" extension should be compared with the existing "kind" extension
